TorchSharp/TorchSharp.csproj

```diff
@@ -44,9 +45,18 @@
     <Compile Remove="T4Template.cs" />
     <Compile Remove="Handle.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Numerics.Tensors" Version="0.1.0-preview2-181029-2" />
```

Comment:
Consdier using version `0.1.0-preview*`, which should resolve to the newest version available.

Reply:
Oh I didn't know. Fixed.